templates/weavescope/0/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+weavescope-probes:
+  image: weaveworks/scope:0.11.1
+  privileged: true
+  net: host
+  pid: host
+  labels:
+    io.rancher.scheduler.global: true
+    io.rancher.container.dns: true
+  links:
+    - weavescope-apps:weavescope
+  ports:
+  - "4040:4040"
+  volumes:
+  - "/var/run/docker.sock:/var/run/docker.sock"
+  tty: true
+  command:
+  - "--probe.docker"
+  - "true"
+  - "--no-app"
+  - "weavescope"
```

Comment:
It will get the DNS entry because of the link. There will be a slight lag of few seconds while the resolv.conf is setup inside the container, will the app die if it can't resolve? or will it retry? 

Also, yes please squash :)